backend/src/openarchiefbeheer/destruction/api/viewsets.py

```diff
@@ -373,8 +373,13 @@ def mark_ready_review(self, request, *args, **kwargs):
 
         return Response(status=status.HTTP_201_CREATED)
 
-    @action(detail=True, methods=["post"], name="abort-destruction")
-    def abort_destruction(self, request, *args, **kwargs):
+    @action(detail=True, methods=["post"], name="abort")
+    def abort(self, request, *args, **kwargs):
```

Comment:
The API schema on line 210 needs also updating

Reply:
- [ ] Update API schema